app/views/handlers/individual_response.py

```diff
@@ -0,0 +1,335 @@
+                }
+            ],
+        },
+    }
+
+    def __init__(self, schema, questionnaire_store, language, form_data, list_item_id):
+        super().__init__(
+            self.block_definition,
+            schema,
+            questionnaire_store,
+            language,
+            form_data,
+            list_item_id,
+        )
+
+    def handle_get(self):
+        previous_location_url = url_for(
+            "individual_response.request_individual_response",
+            list_item_id=self._list_item_id,
+        )
+
+        return render_template(
+            "individual_response/how",
+            language=self._language,
+            content=self.get_context(),
+            previous_location_url=previous_location_url,
+        )
+
+
+class IndividualResponsePostAddressConfirmHandler(IndividualResponseHandler):
+    block_definition = {
+        "type": "IndividualResponse",
+        "question": {
+            "type": "Question",
+            "id": "individual-response-post-confirm",
+            "title": {
+                "text": lazy_gettext(
+                    "Do you want to send an individual access code to <em>{person_name}</em> by post?"
+                ),
+                "placeholders": [
+                    {
+                        "placeholder": "person_name",
+                        "transforms": [
+                            {
+                                "arguments": {
+                                    "delimiter": " ",
+                                    "list_to_concatenate": {
+                                        "identifier": ["first-name", "last-name"],
+                                        "source": "answers",
+                                    },
+                                },
+                                "transform": "concatenate_list",
+                            }
+                        ],
+                    }
+                ],
+            },
+            "description": {
+                "text": lazy_gettext(
+                    "<p>A letter with an indivdual access code will be sent to:</p>"
+                    "<strong>Individual Resident</strong><br>"
+                    "{display_address}"
+                ),
+                "placeholders": [
+                    {
+                        "placeholder": "display_address",
+                        "value": {
+                            "identifier": "display_address",
+                            "source": "metadata",
+                        },
+                    }
+                ],
+            },
+            "guidance": {
+                "contents": [
+                    {
+                        "description": lazy_gettext(
+                            "For security reasons we can only send access codes to an "
+                            "unnamed resident at the registered household address"
+                        )
+                    }
+                ]
+            },
```

Comment:
We need to get confirmation of the design, because they have changed since, the property now uses:
![image](https://user-images.githubusercontent.com/35296336/86612150-95175780-bfa7-11ea-809d-65651d3fea4a.png)

**It is hard to review against a design that is changing. Need confirmation of the design to review this against. As mentioned above, the guidance has changed, the address is no longer piped in the prototype but is here etc. So I can't review it properly without knowing what it should be.**

Reply:
I've asked @jrbarnes9 for confirmation on this

Reply:
Confirmed that this  is the new design. I'll update in this ticket.

Reply:
Does this apply to all the new screens?. I think this was the one stood as a very different.

Reply:
Looks like this has changed again - I will push the updated change shortly